core/master_utils.c

```diff
@@ -747,9 +747,24 @@ int uwsgi_respawn_worker(int wid) {
 		pthread_mutex_lock(&uwsgi.threaded_logger_lock);
 	}
 
+
+	// NOTE: we need to acquire the import lock before forking as is done in os.fork(). We can release it after forking.
```

Comment:
this comment references `os.fork()` which is Python specific, not sure it makes sense being right here